src/App.tsx

```diff
@@ -362,280 +287,242 @@ const formatPercentage = (a: number, b: number): string =>
     const quotient = b === 0 ? 0 : a / b
     return `${(Math.round(quotient * 10000) / 100).toFixed(2)}%`
   }
-const getPartyById = (id: string) =>
-  election.parties.find((party) => party.id === id)
-const sumCandidateVotes = (candidates: ResultsCandidates): number =>
-  Object.keys(candidates).reduce((sum, key) => sum + candidates[key], 0)
-const datesAreOnSameDay = (first: Date, second: Date): boolean =>
-  first.getFullYear() === second.getFullYear() &&
-  first.getMonth() === second.getMonth() &&
-  first.getDate() === second.getDate();
-
-// pre-election || during-election || post-election
+
+const sumCompressedTallies = (compressedTallies: CompressedTally[]): CompressedTally =>
+  compressedTallies.reduce(
+    (sum, tally) => sum.length === 0
+      ? tally
+      : sum.map(
+        (contest, contestIndex) => contest.map(
+          (option, tallyIndex) => option + tally[contestIndex][tallyIndex]
+        )
```

Comment:
we had no existing way in `utils` to sum up tallies? I want to make sure we're not doing tallying in more places than we need to.

Reply:
Asked @carolinemodic, we decided this was the best way.

Reply:
Yeah with the old format SerializedTally and combining tallies from multiple BMD we did have a function for combining those tallies but with the removal of that type and feature it got deleted so we don't really have anything anymore.